admin_guide/diagnostics_tool.adoc

```diff
@@ -115,3 +115,55 @@ A client with *cluster-admin* access available (for any user, but only the
 current master) should be able to diagnose the status of infrastructure such as
 nodes, registry, and router. In each case, running `oc adm diagnostics` looks
 for the client configuration in its standard location and uses it if available.
+
+[[additional-cluster-health-checks]]
+== Additional Diagnostic Checks
+
+Some additional diagnostic checks are available through the https://hub.docker.com/r/openshift/openshift-ansible[openshift-ansible]
+Docker image. These checks are a set of diagnostic tasks meant to be run against a running OpenShift cluster.
```

Comment:
Couple of things.

- At the moment the checks are a single Ansible task. "Set of tasks" might be a little confusing here.
- Also I want it to be really clear here that this is Ansible-based and you need an inventory and can't just run this against minishift etc.

Suggestion:
"These checks belong to a diagnostic task meant to be run against the Ansible inventory file for a deployed OpenShift cluster."